scripts/trigger-metrics.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+
+import os
+import json
+import requests
+import sys
+
+def TriggerPipeline(token, commit, ci_ref):
+    url = "https://circleci.com/api/v2/project/github/mapbox/mobile-metrics/pipeline"
+
+    headers = {
+        "Content-Type": "application/json",
+        "Accept": "application/json",
+    }
+
+    data = {
+        "parameters": {
+          "run_ios_navigation_benchmark": True,
+          "target_branch": commit,  
+          "ci_ref": ci_ref,
+        }
+    }
+
+    # Branch in mobile-metrics repo if you want to trigger a custom pipeline
+    # data["branch"] = "test"
+
+    response = requests.post(url, auth=(token, ""), headers=headers, json=data)
```

Comment:
I am not too familiar with Python, but will this correctly send `token=${MOBILE_METRICS_TOKEN}` parameter? I see there `""` which confuses me.

Reply:
https://circleci.com/docs/api/v2/
> The username should be set as the circle-token value, and the password should be left blank

This code works for sure, just a strange CircleCI API.